.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -1,5 +1,5 @@
-name: Bug Report
-description: Create a report to help us improve
+name: Report a bug 🪳
```

Comment:
i hate this bug emoji, please don't make me look at it.

🐛 is better (`:bug:`)